### PR TITLE
Add setting for pickable WMS layers.

### PR DESCRIPTION
**Pull request by vdubr**
Let buble onLayerClick handler if its defined on Map component

### CODE REVIEW

File: src/components/maps/Map.js [hunk 1]
```diff
@@ -86,14 +86,30 @@ const mapDispatchToPropsFactory = () => {
...
 				},
-				onLayerClick: (mapKey, layerKey, selectedFeatureKeys) => {
+				onLayerClick: (
+					mapKey,
+					layerKey,
+					selectedFeatureKeys,
+					position,
+					options
+				) => {
 					dispatch(
```
Comment (tmdmc) [inline src/components/maps/Map.js:96]: Shouldn't the dispatch also be conditional? Do we always want both dispatch and bubbling above?

Comment (vdubr) [reply src/components/maps/Map.js:96]: I did not want to change API, but this handler onLayerClick is in "controlled" section, so I think it should dispatch onLayerClick every time.

Comment (tmdmc) [reply src/components/maps/Map.js:96]: The dispatch is Action.maps.setLayerSelectedFeatureKeys - should that be always done even when there is another onLayerClick method supplied?

Comment (vdubr) [reply src/components/maps/Map.js:96]: That's better.  Hmm, I agree.

**Review by tmdmc — APPROVED** [review]

**Pull request merged**